### PR TITLE
complete task (TA frameworks basics, 12755)

### PR DESCRIPTION
**Pull request by Irina-N23**
Task #12755 has been completed.
Just check whether everything is correct in it!

### CODE REVIEW

File: test/specs/angularjs-scenario.spec.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        await homePage.learnButton.clickOn();
+        await homePage.tutorialLink.clickOn();
+        const tutorialUrl = await homePage.getCurrentUrl();
+        expect(tutorialUrl).to.be.eql("https://docs.angularjs.org/tutorial");
+    });
+
+
+    it("should find an article about \"bind\"", async () => {
+        await tutorialPage.searchField.inputText("bind");
+        await tutorialPage.wait(3000);
```
Comment (SiarheiRylach) [inline test/specs/angularjs-scenario.spec.js]: this practice should avoided.

File: test/utilities/pageObjects/evernote/home-page.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        this.URL = "https://evernote.com/";
+        this.loginButton = new Element("Login Button", by.xpath("//div[@class="
+                                               + "'top']//a[text()='Log In']"));
+    }
+
+    openPage() {
+        return super.openPage(this.URL);
+    }
+
+    async getTitleOfPage() {
```
Comment (SiarheiRylach) [inline test/utilities/pageObjects/evernote/home-page.js]: not necessary async

File: test/utilities/pageObjects/base-element.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+
+class Element {
+    constructor(name, locator) {
+        this.name = name;
+        this.locator = locator;
+        this.element = element(locator);
+    }
+
+    TIMEOUT = 40000;
```
Comment (SiarheiRylach) [inline test/utilities/pageObjects/base-element.js]: const?

File: test/config/protractor-config.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    framework: "mocha",
+    mochaOpts: {
+        reporter: "mochawesome",
+        timeout: 70000
+    },
+
+    specs: "../specs/*.spec.js",
+
+    capabilities: {
+        "browserName": yargs.browser || "chrome",
```
Comment (SiarheiRylach) [inline test/config/protractor-config.js]: not necessary " symbols

File: test/specs/angularjs-scenario.spec.js [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+        await tutorialPage.searchField.element.sendKeys(Key.ENTER);
```
Comment (SiarheiRylach) [inline test/specs/angularjs-scenario.spec.js]: better to wrap with smth in Element. API methods should not be used on test layer.

File: test/specs/evernote-scenario.spec.js [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+                                                          .getTextFromElement();
+        expect(emailLinkText).to.be.eql(email);
+    });
+
+
+    it("should find a chosen note", async () => {
+        await allNotesPage.searchField.inputText("To-Do", Key.ENTER);
+        const foundResults = await allNotesPage.searchResults
+                                                          .getTextFromElement();
+        await browser.executeScript("console.log(`Search results are "
```
Comment (SiarheiRylach) [inline test/specs/evernote-scenario.spec.js:42]: why?

File: test/specs/angularjs-scenario.spec.js (hunk 1 above)
Comment (Irina-N23) [reply test/specs/angularjs-scenario.spec.js]: Thank you for your feedback! I have invented a better way to implement that part of the code.

File: test/utilities/pageObjects/evernote/home-page.js (hunk 2 above)
Comment (Irina-N23) [reply test/utilities/pageObjects/evernote/home-page.js]: Ok, that was removed from both home pages

File: test/utilities/pageObjects/base-element.js (hunk 3 above)
Comment (Irina-N23) [reply test/utilities/pageObjects/base-element.js]: Yes, I considered that variable was a const in the class. I've changed it to the usual variable. Is it correct now?

File: test/config/protractor-config.js (hunk 4 above)
Comment (Irina-N23) [reply test/config/protractor-config.js]: thank you! Removed!)

File: test/specs/angularjs-scenario.spec.js (hunk 5 above)
Comment (Irina-N23) [reply test/specs/angularjs-scenario.spec.js]: ok, thanks! I have corrected that part of my code.

File: test/specs/evernote-scenario.spec.js (hunk 6 above)
Comment (Irina-N23) [reply test/specs/evernote-scenario.spec.js:42]: The only purpose of that console.log() is to demonstrate the work of the method browser.executeScript(). 
Shall I delete it or write another implementation of browser.executeScript()?

Comment (SiarheiRylach) [reply test/specs/evernote-scenario.spec.js:42]: no, it's just a question. But keep in mind that you cannot open devtools to see it during test execution.

Comment (Irina-N23) [reply test/specs/evernote-scenario.spec.js:42]: ok, thank you)

File: test/utilities/pageObjects/base-element.js (hunk 3 above)
Comment (SiarheiRylach) [reply test/utilities/pageObjects/base-element.js]: yep

**Review by SiarheiRylach — APPROVED** [review]

**Pull request merged**